sambacc/ctdb.py

```diff
@@ -704,6 +704,42 @@ def check_nodestatus(cmd: samba_cmds.SambaCommand = samba_cmds.ctdb) -> None:
     samba_cmds.execute(cmd_ctdb_check)
 
 
+def _read_command_pnn(cmd: samba_cmds.SambaCommand) -> typing.Optional[int]:
```

Comment:
I am not a big fan of this mixed int-or-None return value. I would prefer a more explicit typing; something like `tuple[int, bool]`. But I guess its just a matter of style.

Reply:
Yeah, sorry but I find your suggestion less idiomatic. Hope you don't mind.